README.md

```diff
@@ -196,7 +201,7 @@ Download minetest_game, without using Git:
 Build a version that runs directly from the source directory:
 
     cmake . -DRUN_IN_PLACE=TRUE
-    make -j$(nproc)
+    make -j$(nproc 2>/dev/null || sysctl -n hw.ncpu 2>/dev/null)
```

Comment:
I'd rather add some explanatory text explaining to replace `$(nproc)` with the cpu count instead of this.

Reply:
as a side note, if you do `make -j` without nproc it uses all the available cores nowadays :)